### ISSUE TITLE
Support high contrast mode in chrome without a plugin

### ISSUE THREAD

**Issue by wkeese** [carbon-design-system/carbon#8413]
### Summary

Although Carbon supports Windows High Contrast Mode in Chrome [with a Chrome plugin](https://github.com/carbon-design-system/carbon/blob/master/docs/on-windows-hcm-support.md), you should be able to support it on native Chrome.

This can be done (in theory) by using the [forced-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors) media query instead of the `prefers-contrast` and `-ms-high-contrast` media queries.

### Justification

Easier for vision-impaired people to use Carbon web apps if they don't need to install a plugin.

Also,  according to MDN, `forced-colors` is a web standard whereas `prefers-contrast` hasn't been approved, and of course, `-ms-high-contrast` is Microsoft-specific.

**Comment by dakahn** [comment]
The plugin model that Chrome used for years is more initial friction, but ultimately more control as well. We've found in testing that low-vision users utilizing enhanced contrast modes will often have to toggle the feature off and on as they browse depending on if a site is white text on a black background or black text on a white background (or in the case of carbondesignsystem.com -- both 😨). Flicking WHCM off/on is a pretty jarring and slow experience taking almost 5 seconds on my PC specced for gaming.

That said it looks like native support for WHCM in Chromium browsers without Google's accessibility extensions has landed 👍🏽 (it's been in developments since 2019, I think). (tested in Edge and Chrome latest on Windows 10.0.19041)

I'll update those docs to reflect the new native non-plugin based support. Closing as resolved.

**Comment by wkeese** [comment]
Huh?   This ticket is about ACTUALLY supporting it, not just CLAIMING it's supported.   Supporting it in Carbon, not The Washington Post website.  It doesn't (fully) work in Carbon for the reason I listed above.   Here's an example, Chrome on the left (broken because it's missing the chevrons) and Edge on the right.

<img width="1294" alt="Screen Shot 2021-04-16 at 8 08 20" src="https://user-images.githubusercontent.com/69599/114948990-167f0d00-9e8b-11eb-8ebd-407285484283.png">

**Comment by dakahn** [comment]
☮️ ✌🏽☮️ ✌🏽☮️ ✌🏽☮️ ✌🏽 
Was simply demonstrating that native WHCM support is in which I misunderstood as the point of this ticket. Figured you were saying "hey, we shouldn't have to use a plugin for high contrast mode on Windows in Chromium browsers, here's a few ways we might be able to get it to work".

Nowhere in the original issue description did you report a high contrast mode violation, but I can recreate the one you pointed out. I'll open a ticket to track that work and do a quick audit of native support. If there are enough violation's I'll create an umbrella epic to track the work and tag you in it. 

Thanks!

**Comment by wkeese** [comment]
OK, thanks.   I'm expecting though that you'll end up with dozens of tickets -- one for every place you currently use `prefers-contrast` -- which seems like overkill for what should just essentially be a global search-and-replace.   That's why I thought this one ticket would be sufficient.

**Comment by tw15egan** [comment]
WIP PR: #8489
